pyHS100/pyHS100.py

```diff
@@ -305,6 +344,39 @@ def location(self):
 
         return loc
 
+    @location.setter
+    def location(self, location=(0.0, 0.0)):
+        """
+        Sets new location
+
+        :param tuple location: (float(latitude), float(latitude))
+        :raises SmartPlugException: on error
+        """
+
+        latitude, longitude = location
+
+        if latitude < 0:
+            latitude = -latitude
+        if longitude < 0:
+            longitude = -longitude
+
```

Comment:
ok so that needs to be removed and the user does whatever. and if it is wrong then that's on them. LOL